<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StorageReserveWidget</class>
 <widget class="QWidget" name="StorageReserveWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>166</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="main_frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>921</width>
     <height>151</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="left_frame_2">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>581</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="left_frame">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="BaseTableView" name="table_widget">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="right_frame">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>-1</y>
      <width>206</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="sum_frame">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="sum_layout_2">
        <property name="margin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="buttons_frame">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="buttons_layout_2">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
        <item>
         <widget class="QPushButton" name="plus_button">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>55</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>55</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Добавить позицию</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="minus_button">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>55</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>55</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Вернуть позицию в синий экран</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="destroy_buttons_layout"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="move_buttons_layout">
          <item>
           <widget class="QPushButton" name="clear_button">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>25</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Вернёть всё в синий экран</string>
            </property>
            <property name="text">
             <string>Очистить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="close_button">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>25</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Закрыть</string>
            </property>
            <property name="text">
             <string>Закрыть</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="tobase_button">
          <property name="minimumSize">
           <size>
            <width>55</width>
            <height>55</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>55</width>
            <height>55</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Отправить позиции в базу</string>
          </property>
          <property name="text">
           <string>В базу</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BaseTableView</class>
   <extends>QTableView</extends>
   <header location="global">common/basetableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
